### ISSUE TITLE
No Status Labels Appear Next to Post Title in List

### ISSUE THREAD

**Issue by bbertucc** [Automattic/Edit-Flow#395]
After I enable the custom status feature, all the status labels disappear from the post bowser. 
![screen shot 2017-05-30 at 3 47 59 pm](https://cloud.githubusercontent.com/assets/46652/26604358/b7715cca-454f-11e7-9e50-37e18ac27b64.png)

Edit flow is the only plugin I have active. As soon as I disable the plugin, the status labels reappear.

Any help would be much appreciated.

**Comment by raduconst** [comment]
@bbertucc It seems that this element is hidden on purpose because a custom column displays the status. When using the plugin, there's a new column called _Status_ that displays this info:

<img width="707" alt="screenshot at jan 17 19-24-42" src="https://user-images.githubusercontent.com/6512220/35056952-177acf66-fbbc-11e7-82b6-068fd46955b1.png">

Could you check this out and see if you can see this column on your side, please?

**Comment by raduconst** [comment]
@bbertucc Do you think that it would be helpful to make the status label appear next to the post title when the _Status_ box from _Screen Options_ area is unchecked?
From this:

<img width="1439" alt="screenshot at jan 20 16-10-08" src="https://user-images.githubusercontent.com/6512220/35184370-4f941b08-fdfd-11e7-872a-9188dd4fd9a2.png">

to this:

<img width="1439" alt="screenshot at jan 20 16-16-27" src="https://user-images.githubusercontent.com/6512220/35184372-56978f0c-fdfd-11e7-881b-3d01886a66ca.png">

I think that I can work on something like this if you think that it would be helpful for publishers.

**Comment by sboisvert** [comment]
@raduconst Love the suggestion! Good use of screenshots!

**Comment by raduconst** [comment]
I had a second thought about this and here's how I see things:
1. If the _custom statuses_ (Edit Flow → Custom Statuses → Options → Use on these post types → Posts) are used:
- on the _Post_ section, if the _Status_ checkbox is checked, the Status column should be visible and the status labels should not be visible;
- on the _Post_ section, if the _Status_ checkbox is unchecked, the Status column should be visible and the status labels should not be visible;
2. If custom post status is not used for the posts, the labels are back to the Posts page (/wp-admin/edit.php).

Basically, not that I think again about this, there's no point on hiding the Status column but display the labels. When unchecking the Status box, the main reason for using the option is not to see the statuses at all, not move them. The only problem that remains is the dash when at the end of the post title that's not removed (and I'll fix this through a PR).

**Comment by WPprodigy** [comment]
I think this could be re-visited.

To better adapt to core behavior, the `post states` are the best place to show the status. The column could be done away with completely. Columns become expensive real-estate on big site with lots of information, and this particular information fits well into an already-established UI behavior in core.
